grumphp.yml

```diff
@@ -27,6 +27,7 @@ parameters:
         - /^storage\/(.*)/
         - /^public\/(.*)/
         - /^resources\/(.*)/
+        - /^database\/(.*)/
```

Comment:
This pattern does seem to be correct. Do files staged in any of the other directories trigger Grum?

Reply:
this was fixed here: https://github.com/indiehd/web-api/commit/52b39e6ce87d4bc042bb6032bca2c151c6986b0d